windows.js

```diff
@@ -149,8 +122,9 @@ export function addVCVARSEnv() {
   let newPathEntries = undefined
   for (let [k, v] of newEnv) {
     if (process.env[k] !== v) {
-      if (k === 'Path') {
-        newPathEntries = v.replace(process.env['Path'], '').split(path.delimiter)
+      if (/^Path$/i.test(k)) {
+        const newPathStr = v.replace(`$(path.delimiter)${process.env['Path']}`, '')
```

Comment:
`$(path.delimiter)` => `${path.delimiter}` right?

Reply:
https://github.com/ruby/setup-ruby/pull/83

Reply:
I checked, that Path was doubled with that, before:
https://github.com/ruby/setup-ruby/runs/1066490911?check_suite_focus=true
(e.g., `mysql` is twice in Path)
after:
https://github.com/ruby/setup-ruby/pull/83/checks?check_run_id=1066493565